### ISSUE TITLE
RFP - Office 2007

### ISSUE THREAD

**Issue by valentind44** [chocolatey/chocolatey-package-requests#390]

**Comment by pauby** [comment]
@valentind44 You need to provide additional information. Office 2007 can already be install silently but doesn't have a public source so is not a good candidate for a Community Repository package.

**Comment by valentind44** [comment]
That's too bad, sorry. Direct links from microsoft for other office editions can be obtained here https://tb.rg-adguard.net/public.php .

**Comment by valentind44** [comment]
It generates download links from Microsoft servers.

**Comment by pauby** [comment]
It won't generate any links for Office 2007 in French or English (and I'm not trying any more).

**Comment by valentind44** [comment]
Yes, sorry. I provided the link because it works for other Office versions like Office 2013. It used to work for office 2007 but microsoft doesn't support it anymore so there are no more links available. I provided the link for informative purpose and in case a package could be made for other versions of Office.
Doing more research I've see that https://chocolatey.org/packages/OfficeProPlus2013 is already available anyway, nevermind. It uses Office deployment tool but this method doen't work for Office 2007, too bad.
